example/1_getstarted.html

```diff
@@ -13,10 +13,14 @@
   </div>
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/87/three.min.js"></script>
+  <!--<script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/87/three.module.js"></script>-->

```

Comment:
three.module.js is for bundler and Unnecessary for "static JS". Could you remove this line please?